hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/SlashEncodedMonthPartitionValueExtractor.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.hudi.hive;
+
+import org.apache.hudi.sync.common.model.PartitionValueExtractor;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The SlashEncodedMonthPartitionValueExtractor class is a custom implementation of the
+ * PartitionValueExtractor interface to support extracting partition values from paths
+ * in the format 'yyyy/mm'.
+ */
+public class SlashEncodedMonthPartitionValueExtractor implements PartitionValueExtractor {
```

Comment:
The functionality of this extractor is covered by `SinglePartPartitionValueExtractor`.  Is `SinglePartPartitionValueExtractor` not enough in your use case?

Reply:
Maybe not, the user actually wants the data format with `yyyy/mm` and ignores the datetime.

Reply:
I understand that.  `SinglePartPartitionValueExtractor` serves the same purpose by transforming `yyyy/mm` to `yyyy-mm`.

Reply:
The user want `yyyy/mm` instead.

Reply:
Not following, the logic and test show that `yyyy/mm` is transformed to `yyyy-mm` during extraction.

Reply:
@banank1989 could you clarify?